code/__DEFINES/mobs.dm

```diff
@@ -279,6 +279,7 @@
 //MINOR TWEAKS/MISC
 #define AGE_MIN				17	//youngest a character can be
 #define AGE_MAX				85	//oldest a character can be
+#define AGE_MINOR			21  //legal age of space drinking and smoking
```

Comment:
pls indent this one more

Reply:
its indented to be equal on the file, no idea why it looks like this on files changed